duties/constants/program.py

```diff
@@ -12,3 +12,8 @@
 GRACEFUL_KILLER = GracefulKiller()
 THRESHOLD_TO_INFORM_USER_FOR_WAITING_PERIOD = 5000
 NOT_ALLOWED_CHARACTERS_FOR_VALIDATOR_PARSING = [".", ","]
+NUMBER_OF_VALIDATORS_PER_REST_CALL = 300
```

Comment:
is this client specific or part of beacon api?

Reply:
It looks like this is an important question. When I started development I had this issue for sure and realized that there is a max. around 300-400 validators which I could add to a specific REST call. It seems this restriction doesn't exist anymore or was improved a lot.

I tested with `lighthouse v3.4.0` and `teku v23.1.1` and here I could increase the number of validators per REST call up to 7k. `eth-duties` crashed with 8k validators per call. But I can't say whether this is a restriction of the beacon api spec itself or of the REST implementation. I would rather assume that it is due to the latter. Taking this into account I increased the number of validators per REST call to 1000.

Thanks so much for your useful input :pray: .

Reply:
Just as a side note. I tested now 4/5 consensus clients and replaced the `caveat` section in the README with a consensus client compatibility overview which might be more useful.